examples/python/open_photogrammetry_format/main.py

```diff
@@ -77,8 +77,13 @@ def __init__(self, path: Path, log_as_frames: bool = True) -> None:
         log_as_frames : bool, optional
             Whether to log the cameras as individual frames, by default True
         """
+        import os
+
         self.path = path
-        self.project = resolve(load(str(path)))
+        # TODO(Pix4D/pyopf#6):
```

Comment:
We can afford the extra bytes to make it easier for the reader to find the issue

```suggestion
        # TODO(Pix4D/pyopf#6): https://github.com/Pix4D/pyopf/issues/6
```